releases/release-1.33/release-notes/maps/pr-129790-map.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 129790
+releasenote:
+  text: Fixed the bug where Events fail to be created when the referenced object name
+    is not a valid Event name. Now, a UUID is used as the name instead of the referenced
+    object name and the timestamp suffix.
```

Comment:
```suggestion
  text: Fixed the bug where Events failed to be created when the referenced object name
    was not a valid Event name. Now, a UUID is used as the name instead of the referenced
    object name and the timestamp suffix.
```